scripts/main/album.js

```diff
@@ -493,6 +494,7 @@ album.setDescription = function (albumID) {
 			},
 		},
 	});
+	$("#album_desc").html(oldDescription).select();
```

Comment:
I think it's because the value of a `textarea` doesn't really get interpreted as HTML; it is simply copied into the input box in a "row" form. I'll try your suggestion of using `.text` but I'm guessing that may not work correctly, actually (won't the escaped sequences become visible in the textarea box?).